bindings/pyroot/pythonizations/src/CPPInstancePyz.cxx

```diff
@@ -83,13 +83,19 @@ PyObject *op_reduce(CPPInstance *self, PyObject * /*args*/)
    if (s_bfClass == self->ObjectIsA()) {
       buff = (TBufferFile *)self->GetObject();
    } else {
+      auto className = Cppyy::GetScopedFinalName(self->ObjectIsA());
+      if (className.find("__cppyy_internal::Dispatcher") == 0) {
+         PyErr_Format(PyExc_IOError, "cannot stream Python objects whose class derives from a C++ class. "
```

Comment:
Maybe specify that it cannot be streamed *by default* and that the custom `__reduce__` function can make the serialization work. Also maybe something like "Refer to the python pickle package standard documentation for help".

Reply:
Done!